docs/dashboard.md

```diff
@@ -6,7 +6,7 @@ show the status of SC4SNMP tasks.
 ## Presetting
 
 !!! info
-    Dashboard is compatible starting from version 1.11.0
+    Dashboard is compatible starting from version **1.11.0** and requires the `logLevel` set at least to **INFO**.
```

Comment:
`at least INFO` - Info it's minimal level, it should be `logLevel set to **INFO**`